frontend/packages/console-dynamic-plugin-sdk/docs/api.md

```diff
@@ -1903,6 +1903,7 @@ A basic lazy loaded YAML editor with hover help and completion.
 | `toolbarLinks` | Array of ReactNode rendered on the toolbar links section on top of the editor. |
 | `onChange` | Callback for on code change event. |
 | `onSave` | Callback called when the command CTRL / CMD + S is triggered. |
+| `ref` | React reference to monaco editor. For more details, please, visit https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.IStandaloneCodeEditor.html. |
```

Comment:
Can we describe the structure of the object seeing as it is `{ editor?: IStandaloneCodeEditor}`

Reply:
The structure is a little complex. There are, like, hundreds of methods. Do you have any suggestions on how should I describe the structure? Maybe I can say that it can be used to call methods to control the editor?

Reply:
I wanted to describe that `ref` will have the shape: `{editor?: ...}`. It's not important to describe `IStandaloneCodeEditor` here. The link can remain.